cli/borges/main.go

```diff
@@ -19,12 +20,26 @@ var (
 )
 
 type cmd struct {
-	Queue string `long:"queue" default:"borges" description:"queue name"`
+	Queue    string `long:"queue" default:"borges" description:"queue name"`
+	LogLevel string `short:"" long:"loglevel" description:"max log level enabled" default:"info"`
```

Comment:
I think this will add the log level and log file argument to the version command, but the version command it is not calling ChageLogLevel.

We should either call ChangeLogLevel from the version command or include these arguments only in the consumer and producer commands.

I think the second solution is better than the first.